Cargo.toml

```diff
@@ -12,10 +12,11 @@ license = "MIT"
 edition = "2021"
 
 [features]
+default = ["clap"]
```

Comment:
No need to make it default then.

Reply:
You're ok to break `cargo b --bin minifier`?